### PR TITLE
SliTaz support added

### PR DESCRIPTION
**Pull request by nilesr**
## Description

I've tested this on a strata of SliTaz with all the other strata disabled, but if you need more testing I'd be happy to spin up a virtual machine. Also let me know if you want the coding style or position in the OS detection order or anything changed.

## Features

- Add support for [SliTaz](http://slitaz.org/en)

## TODO

- Test on an actual installation of slitaz

Previous tests

Yes I know my shell doesn't exist, that's because I logged in with fish, then exec'd to SliTaz's installation of bash, then disabled the strata that fish exists in.

Yes I know it says "Fedora" but that's because lsb_release is reporting Fedora, not a bug in the neofetch code.

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -476,6 +479,9 @@ get_packages() {
             type -p crew >/dev/null && \
                 packages="$((packages+=$(ls -l /usr/local/etc/crew/meta/*.filelist | wc -l)))"
 
+            type -p tazpkg >/dev/null && \
+                packages="$(($(tazpkg list|wc -l)-6))"
```
Comment (dylanaraps) [inline neofetch]:
```sh
packages="$((packages+=$(tazpkg list | wc -l) - 6))"
```

File: neofetch [hunk 2]
```diff
@@ -93,6 +93,9 @@ get_distro() {
                     "tiny") distro="${distro//version*}" ;;
                 esac
 
+            elif type -p tazpkg >/dev/null; then
+                distro="SliTaz $(cat /etc/slitaz-release)"
```
Comment (dylanaraps) [inline neofetch]:
```sh
distro="SliTaz $(< /etc/slitaz-release)"
```

File: neofetch [hunk 3]
```diff
@@ -3150,6 +3156,11 @@ get_distro_colors() {
             ascii_file="slackware"
         ;;
 
+        "SliTaz"*)
+            set_colors 3 3
```
Comment (dylanaraps) [inline neofetch]: Indentation doesn't match the rest of the script (4 spaces).

File: neofetch [hunk 4]
```diff
@@ -3150,6 +3156,11 @@ get_distro_colors() {
...
+            ascii_file="slitaz"
```
Comment (dylanaraps) [inline neofetch]: Indentation doesn't match the rest of the script (4 spaces).

File: neofetch (hunk 1 above)
Comment (dylanaraps) [inline neofetch]: Indentation doesn't match the rest of the script (4 spaces).

File: neofetch (hunk 2 above)
Comment (dylanaraps) [inline neofetch]: Indentation doesn't match the rest of the script (4 spaces).

**Comment by dylanaraps** [comment]
I can fix indentation afterwards if you'd like.

**Comment by nilesr** [comment]
I've corrected the style changes but my editor is saying there are no tabs in the whole file. I thought I did indent with four spaces.

**Comment by dylanaraps** [comment]
LGTM now :+1:

**Comment by dylanaraps** [comment]
Just waiting for Travis before merge.

File: neofetch [hunk 5]
```diff
@@ -93,6 +93,9 @@ get_distro() {
                     "tiny") distro="${distro//version*}" ;;
                 esac
 
+            elif type -p tazpkg >/dev/null; then
+                distro="SliTaz $(< /etc/slitaz-release)"
```
Comment (konimex) [inline neofetch]: Is this really necessary? Did `/etc/os-release` gives (any) output? If this doesn't exist, there's already a last fallback.

```sh
for release_file in /etc/*-release; do
 distro+="$(< "$release_file")"
done
```

Comment (dylanaraps) [inline neofetch]: The problem is lsb_release showing "Fedora", this is the same issue as MX and AntiX. :/

Comment (konimex) [inline neofetch]: Oops. Didn't see the second screenshot. Was focused on 'strata' so I thought it was a Bedrock Linux instance.

While we're at that, how would neofetch work in a Bedrock Linux instance...

Comment (dylanaraps) [inline neofetch]: It probably \*wouldn't\* work, it'd be neat to test it out though.

Comment (nilesr) [inline neofetch]: It is running inside bedrock. Running `brc slitaz brw lsb_release` gives me `fedora (implicit)`. Slitaz by default doesn't come with the `lsb_release` command or `/etc/os-release`

Comment (nilesr) [inline neofetch]: And there is no other release information: http://i.imgur.com/AlQwanK.png

Comment (konimex) [inline neofetch]: Since it has no other release information, I think we can delete those lines since I don't think they are necessary.

Comment (nilesr) [inline neofetch]: If I remove those two lines it just reports `OS: 5.0 x86_64` on the first line of output.

**Comment by konimex** [comment]
Alright, all clear. Travis seems to be good too, so I'm merging this into master, thanks!

Changelogs will be added later since I'm actually in transit.

**Pull request merged**